module-1/src/Components/MovieDeleteForm/MovieDeleteForm.tsx

```diff
@@ -0,0 +1,50 @@
+interface MovieDeleteFormProps {
+  title: string;
+  movie?: Movie | null;
+  closeCallback: (b: boolean) => void;
+}
+export const MovieDeleteForm = ({
+  title,
+  movie,
+  closeCallback,
```

Comment:
Normally `callback` is not used as a part of a name. You can go for onX, handleX or just X.

Reply:
updated